contracts/Paper.sol

```diff
@@ -0,0 +1,77 @@
+pragma solidity ^0.8.4;
+
+contract Paper {
+    enum State {onReview, Approved, Rejected}
+
+    struct FeedBack {
+        string feedback;
+        address reviewer;
+    }
+
+    string public ipfsHash;
+    string[] public fields;
+    State public paperState;
+    address private owner;
+    address[] private reviewers;
+    State[] private reviewStates;
+    FeedBack[] private feedbacks;
+    uint private submitDate;
+    uint private deadlineDate;
```

Comment:
You should use Open Zeppelin library whenever dealing with numbers. It provides guarantees for over/underflows.